content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -34,51 +37,43 @@ For manual deployment or replacement of kube-dns, see the documentation at the
 
 ### Upgrading an existing cluster with kubeadm
 
-In Kubernetes version 1.10 and later, you can also move to CoreDNS when you use `kubeadm` to upgrade
-a cluster that is using `kube-dns`. In this case, `kubeadm` will generate the CoreDNS configuration
+In Kubernetes version 1.21, kubeadm removed its support for `kube-dns` as a DNS application.
+For `kubeadm` v{{< skew currentVersion >}}, the only supported cluster DNS application
+is CoreDNS.
+
+You can move to CoreDNS when you use `kubeadm` to upgrade a cluster that is
+using `kube-dns`. In this case, `kubeadm` generates the CoreDNS configuration
 ("Corefile") based upon the `kube-dns` ConfigMap, preserving configurations for
 stub domains, and upstream name server.
 
-If you are moving from kube-dns to CoreDNS, make sure to set the `CoreDNS` feature gate to `true`
-during an upgrade. For example, here is what a `v1.11.0` upgrade would look like:
-```
-kubeadm upgrade apply v1.11.0 --feature-gates=CoreDNS=true
-```
-
-In Kubernetes version 1.13 and later the `CoreDNS` feature gate is removed and CoreDNS
-is used by default.
-
-In versions prior to 1.11 the Corefile will be **overwritten** by the one created during upgrade.
-**You should save your existing ConfigMap if you have customized it.** You may re-apply your
-customizations after the new ConfigMap is up and running.
-
-If you are running CoreDNS in Kubernetes version 1.11 and later, during upgrade,
-your existing Corefile will be retained.
+## Upgrading CoreDNS
 
-In Kubernetes version 1.21, support for `kube-dns` is removed from kubeadm.
+You can check the version of CoreDNS shipped with Kubernetes and the changes
+made to CoreDNS [here](https://github.com/coredns/deployment/blob/master/kubernetes/CoreDNS-k8s_version.md).
```

Comment:
(nit)
I'd write this as:
```suggestion
You can check the version of CoreDNS that kubeadm installs for each version of
Kubernetes, in the page
[CoreDNS version in Kubernetes](https://github.com/coredns/deployment/blob/master/kubernetes/CoreDNS-k8s_version.md).
```

Reply:
I'd ignore the ‘kubeadm installs’ part because 'kubeadm' is only one of the ways to setup k8s and its plugins.
I'll take your suggestion revising the link anchor. Using 'here' as anchor is not the recommended style.

Reply:
From the linked page:
> This document records the CoreDNS version that was installed by kubeadm with each version of Kubernetes since v1.9.

It is specifically about kubeadm. Other tools might deploy a different cluster DNS solution, or CoreDNS but a different version.

Reply:
Okay, fair enough. I didn't check the page. Will revise.